google.json

```diff
@@ -1,13 +1,200 @@
+        "description": "Have some stuff about google gms: backup.BackupTransportService.\nUseless backup things, without it backup and cloud work.\nMaybe its used for backup settings, but probably not, and if its, I don't think you need it.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.overlay.gmsconfig.gsa",
+        "description": "configures default assistant? And have config to allow disabling assist disclosure?\nI think it doesnt affect on anything.\nSafe to remove if you removed (com.google.android.googlequicksearchbox).",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.overlay.gmsconfig.common",
+        "description": "Useless configurations about webview, location, lists google apps.\neverything works without it.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.overlay.gmsconfig.comms",
+        "description": "Useless configurations dialer, sms, contacts Google(GApps)\neverything works without it.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.overlay.gmsconfig.geotz",
+        "description": "Provides Geolocation Time Zone detection.\nand its used by (com.google.android.overlay.gmsconfig.common)?\nNo effects after remove.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.nearby.halfsheet",
+        "description": "Useless frameworks to wifi connection, usb tethering, auto, usage.\nevery version have random code and app not running in the background.\nlooks like a very useless.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.as.oss",
+        "label": "Private Compute Services",
+        "description": "Its another component of\nAndroid System Intelligence (previously Device Personalization Services)",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.overlay.gmsconfig.asi",
+        "description": "Its another component of\nAndroid System Intelligence (previously Device Personalization Services)",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.android.overlay.modules.captiveportallogin.forframework",
+        "description": "Configs default captiveportallogin. (not needed)\nNo effects after remove.",
+        "removal": "delete"
+    },
+    {
+        "id": "com.google.mainline.telemetry",
+        "description": "Contains data on which versions of modules are installed. Google Play uses this data to determine if updates are available for the modules, and to show which security patch is installed.\nThis module doesn’t contain active code and has no functionality on its own.\nAnyway I wont trust it when adservices are also in mainline.",
+        "web": [
+            "https://www.xda-developers.com/android-project-mainline-modules-explanation/",
+            "https://gitlab.com/W1nst0n/universal-android-debloater/-/issues/27#note_410012436"
+        ],
+        "removal": "replace"
```

Comment:
Why `replace`?

Reply:
idk like seriosly its mainline(security) thing but I see a lot of custom roms dont need these things also "adservices" mainline